### ISSUE TITLE
Improve WYSIWYG Toggle Button

### ISSUE THREAD

**Issue by psy-q** [cosmocode/dokuwiki-plugin-prosemirror#92]
I have feedback from our users that they are afraid to click the "Toggle WYSIWYG editor" button because they don't know what WYSIWYG means. Once explained, it was OK, but it's creating support issues.

Especially for younger people, this acronym has no meaning and just looks scary and long so they avoid it, fearing they could break something if they click the button.

Other, similar software uses phrasing like "visual editor" (my favorite) or "easy editor". What do you think about renaming the button? Or perhaps this button design isn't final yet anyway?

**Comment by splitbrain** [comment]
The current position and styling is currently sub-optimal and was meant as a temporary mechanism only. The button should be redesigned and integrated properly into the interface.

#86 should be fixed together with this issue.

**Comment by solarkraft** [comment]
"Visual Editor" would be in line with Mediawiki terminology and my preferred name. The toggle button _should_ be improved, but much more important than that, IMO, would be having the visual editor on by default (as a normal user I'd be quite afraid of the weird markup).

**Comment by solarkraft** [comment]
A good way of exiting the visual editor would probably be a button in the toolbar (perhaps hidden behind the settings wheel) - since if the visual editor works, you shouldn't ever need to use it, using the text is definitely a power user feature. Conversely it may make sense for the button to be more prominent in the text view for people who opened it by accident to easily get back.

**Comment by solarkraft** [comment]
For my own purposes I restyled the button like this ...

```
/* Make the WYSIWYG button a bit less obtrusive */
.dokuwiki .plugin_prosemirror_useWYSIWYG {
        color: unset;
        background-color: white;
        top: 2.9em;
        right: 2em;
}
```

... which leads to it looking like this (in my theme, with other modifications):

<img width="811" alt="Screen Shot 2021-08-20 at 03 54 29" src="https://user-images.githubusercontent.com/1337470/130166698-beeb9c03-af16-4d9b-9a13-360b5de85a38.png">

**Comment by solarkraft** [comment]
The restyling seen above has landed in #147.
